net-vpn/openfortivpn/openfortivpn-1.23.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools linux-info
+
+DESCRIPTION="Fortinet compatible VPN client"
+HOMEPAGE="https://github.com/adrienverge/openfortivpn"
+SRC_URI="https://github.com/adrienverge/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3-with-openssl-exception openssl"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	dev-libs/openssl:=
+	net-dialup/ppp
+"
+RDEPEND="${DEPEND}"
```

Comment:
I think we need a BDEPEND on virtual/pkgconfig.

Reply:
Should be covered through the newly inherited systemd eclass.

Reply:
If the package _directly_ relies on pkgconfig, we should depend on that explicitly, not rely on something else doing that for us. (The plan is also to remove that use from systemd.eclass anyway.)